package.json

```
@@ -2,6 +2,7 @@
   "name": "pako",
   "description": "zlib port to javascript - fast, modularized, with browser support",
   "version": "2.1.0",
+  "sideeffects": false,
```

According to the docs you linked, the property should be `sideEffects`- note the casing difference.

Suggested change:
```suggestion
  "sideEffects": false,
```

---

I have done the change. @Greenheart Could you please approve?

---

Could you push the change to the same branch as this PR is based on ([JagjitKaur:Pako/tree-shaking](https://github.com/JagjitKaur/pako/tree/Pako/tree-shaking))?

I'm not a maintainer of this project, so I can't decide whether to merge or not.

However, this is consistent with the Webpack docs - hope it helps your project :)